cluster/config-defaults.yaml

```diff
@@ -70,6 +70,14 @@ skipper_ingress_min_replicas: "2"
 {{end}}
 skipper_ingress_cpu: "1000m"
 skipper_ingress_memory: "1500Mi"
+
+# Non empty value enables deployment of specified canary version
+skipper_ingress_canary_internal_version: ""
+
+# Optional canary arguments separated by "[cf724afc]" to allow whitespaces, e.g.:
```

Comment:
Quite a counterintuitive separator :-)
From my side, it looks like it is possible to find something more simple. For example, I see no `;` used in the current skipper cmdline args.
Why don't we use `;` instead of `[cf724afc]` to make it much more readable?

Reply:
We already use `[cf724afc]` as a separator for two other flags that may contains whitespace or any other sequence of characters. `[cf724afc]` is chosen exactly because it is unlikely to appear in the flag value.

If you are curious where it comes from, it is:
```
~$ echo -n skipper | sha256sum | head -c 8
cf724afc
```

Reply:
Okay, fair enough. :-)